components/Counting.vue

```diff
@@ -1,95 +1,192 @@
 <template>
     <section class="counting">
-      <h4>AI Insights delivered & counting </h4>
-      <h2 id="countUp">1234</h2>
+      <div id="countPhrase" >
+        <h4 id="countPhraseWrapper" >AI Insights delivered & counting</h4>
+      </div>
+      <div id="countUp">
+        <h2 id="countUpWrapper" > </h2>
+      </div>
     </section>
 </template>
 
 <script>
 export default {
     mounted() {
-    function animateValue(id, start, end, duration) {
-      var obj = document.getElementById(id);
-      var range = end - start;
-      var minTimer = 50;
-      var stepTime = Math.abs(Math.floor(duration / range));
-
-      stepTime = Math.max(stepTime, minTimer);
-
-      var startTime = new Date().getTime();
-      var endTime = startTime + duration;
-      var timer;
+      var canv = document.createElement('canvas');
```

Comment:
since we're using the newer <code>``</code> (template string) syntax, should we also use `const` or <code>let</code>?

Reply:
We probably should. I'm a little unfamiliar with Vue, and I saw a mix of both `var` and `const` in the existing code. I did have a few issues with the compiler not handling proper class scoping (it did a `var _this = this;` thing that broke a later enclosure) so I am not certain what amount of ES6 is supported by this compiler. We can change this and test that it works, but I don't think it is necessary.

Reply:
as long as theres a reason for it.